client/src/interfaces/responses/board-permission/grant-permission-response.ts

```diff
@@ -0,0 +1,5 @@
+export interface GrantPermissionResponse {
+  permissionStr: string;
```

Comment:
How would you call it then? I wont approve permission code since its not hashed or sth. PermissionStr means its a permission string used for assigning a permission

Reply:
Why? Code doesn't have to be hashed to be called a code.